src/encoded/types/dataset.py

```diff
@@ -181,6 +190,98 @@ def hub(self, request):
         return request.resource_path(self, '@@hub', 'hub.txt')
 
 
+@collection(
+    name='transgenic-enhancer-experiments',
+    unique_key='accession',
+    properties={
+        'title': 'Transgenic enhancer experiments',
+        'description': 'Listing of Transgenic Enhancer Experiments',
+    })
+class TransgenicEnhancerExperiment(
+    Dataset,
+    CalculatedAssaySynonyms,
+    CalculatedAssayTermID,
+    CalculatedAssaySlims,
+    CalculatedCategorySlims,
+    CalculatedTypeSlims,
+    CalculatedObjectiveSlims):
+    item_type = 'transgenic_enhancer_experiment'
+    schema = load_schema('encoded:schemas/transgenic_enhancer_experiment.json')
+    embedded = Dataset.embedded + [
+        'biosample_ontology',
+        'biosamples',
+    ]
+    audit_inherit = [
+        'submitted_by',
+        'lab',
+        'award',
+        'documents.lab',
+    ]
+    set_status_up = [
+        'documents'
+    ]
+    set_status_down = []
+    rev = Dataset.rev.copy()
+    rev.update({
+        'superseded_by': ('TransgenicEnhancerExperiment', 'supersedes')
+    })
+
+    @calculated_property(schema={
+        "title": "Superseded by",
+        "type": "array",
+        "items": {
+            "type": ['string', 'object'],
+            "linkFrom": "InVivoExperiment.supersedes",
+        },
+        "notSubmittable": True,
+    })
+    def superseded_by(self, request, superseded_by):
+        return paths_filtered_by_status(request, superseded_by)
+
+    @calculated_property(schema={
+        "title": "Biosample summary",
+        "type": "string",
+    })
+    def biosample_summary(self, request, biosamples=None):
+        drop_age_sex_flag = False
+        dictionaries_of_phrases = []
+        if biosamples is not None:
+            for bs in biosamples:
+                biosampleObject = request.embed(bs, '@@object')
+                biosample_dict = biosample_summary_dictionary(request, biosampleObject)
+                dictionaries_of_phrases.append(biosample_dict[0])
+                if biosample_dict[1] is True:
+                    drop_age_sex_flag is True
```

Comment:
I would explicitly define 2 variables here - make the code more readable:
"biosample_summary_dictionary = item[0]" and "biosample_drop_age_sex_flag = item[1]", see my next comment for more details.

Reply:
Defined